pom.xml

```diff
@@ -26,7 +26,7 @@
         <munit.input.directory>src/test/munit</munit.input.directory>
         <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
         <munit.extensions.maven.plugin.version>1.0.0-SNAPSHOT</munit.extensions.maven.plugin.version>
-        <munit.version>2.2.1</munit.version>
+        <munit.version>2.2.3-SNAPSHOT</munit.version>
```

Comment:
why snapshot? can't we use 2.2.2? otherwise, we need to wait for munit to release.

Reply:
Ok, 2.2.3-SNAPSHOT has fixes for MTF on 4.3.0. I think that @diegostrubolini would be releasing this first soon.